src/components/SelfieWebCam/SelfieWebCam.js

```diff
@@ -0,0 +1,229 @@
+// NOTE: This is a customized fork from the `vue-web-cam` NPM package (see: https://github.com/VinceG/vue-web-cam) which was required in order to add support for selfie constraints
+
+const props = {
+  width: {
+    type: [Number, String],
+    default: '100%',
+  },
+  height: {
+    type: [Number, String],
+    default: 500,
+  },
+  autoplay: {
+    type: Boolean,
+    default: true,
+  },
+  screenshotFormat: {
+    type: String,
+    default: 'image/jpeg',
+  },
+  selectFirstDevice: {
+    type: Boolean,
+    default: false,
+  },
+  deviceId: {
+    type: String,
+    default: '',
+  },
+  playsinline: {
+    type: Boolean,
+    default: true,
+  },
+  resolution: {
+    type: Object,
+    default: null,
+    validator: (value) => (value.height && value.width),
+  },
+};
+
+const data = function () {
+  return {
+    source: null,
+    canvas: null,
+    wasCamerasListEmitted: false,
+    cameras: [],
+  };
+};
+
+const computed = {
+  hasActiveVideo() {
+    const result = ((this.$refs.video !== null) && this.$refs.video.srcObject);
+    return result;
+  },
+};
+
+const methods = {
+  legacyGetUserMediaSupport() {
+    const result = (constraints) => {
+      const getUserMedia = (navigator.getUserMedia
+        || navigator.webkitGetUserMedia
+        || navigator.mozGetUserMedia
+        || navigator.msGetUserMedia
+        || navigator.oGetUserMedia);
```

Comment:
🤔  just an idea but... WDYT about move this to a function like `getUserMediaFromClientBrowser` or something like that?

Reply:
Good suggestion! Will do